xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -1838,7 +1830,7 @@ dependencies of the targets specified in the `top_level_targets` attribute with
 a matching label will be excluded from the generated project. This overrides any
 targets specified in the `focused_targets` attribute.
 """,
-            default = [],
+            mandatory = True,
```

Comment:
By making these mandatory, we can catch errors of the runner forgetting to set the values.